### PR TITLE
[4] Method call is provided 3 parameters, but the method signature uses 2 parameters

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on f3d59f06565e3302fc9384f825b49be0946a65d2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33953">issues.joomla.org/tracker/joomla-cms/33953</a>.</sub>

**Comment by joomdonation** [comment]
Simple PR. Set to RTC after reviewing.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33953">issues.joomla.org/tracker/joomla-cms/33953</a>.</sub>

**Comment by PhilETaylor** [comment]
> Simple PR. 

haha apparently thats all Im good at :)

**Comment by joomdonation** [comment]
Not really. You are doing good work. We just decided to do that to save time and also get the PR merged quicker :).

File: administrator/components/com_joomlaupdate/src/Model/UpdateModel.php [hunk 1]
```diff
@@ -144,7 +144,7 @@ public function refreshUpdates($force = false)
 		else
 		{
 			$update_params = ComponentHelper::getParams('com_installer');
-			$cache_timeout = $update_params->get('cachetimeout', 6, 'int');
+			$cache_timeout = $update_params->get('cachetimeout', 6);
```
Comment (joomdonation) [inline administrator/components/com_joomlaupdate/src/Model/UpdateModel.php]: Sorry. Do you mind adding int type casting here? It is not required, but having type casing like https://github.com/joomla/joomla-cms/pull/33956/files#diff-0447907ad0a438c7e159d0425b505e3cc873dc72c344bd2742d4263091909655R45 would be better.

Comment (PhilETaylor) [reply administrator/components/com_joomlaupdate/src/Model/UpdateModel.php]: Done.

Comment (joomdonation) [reply administrator/components/com_joomlaupdate/src/Model/UpdateModel.php]: Great. Thanks !

**Comment by Quy** [comment]
Thank you!

**Pull request merged**